src/components/zen-table/zen-table.tsx

```diff
@@ -17,6 +17,18 @@ export class ZenTable {
   /** Table cleanup in progress */
   @Prop({ attribute: 'updating' }) readonly $updating: boolean = false;
 
+  /** Returns array with ids of selected rows (ordered as they appear in the dom) */
+  @Method()
+  async getSelectedRows(): Promise<string[]> {
+    return this.getAllRows()
+      .filter(row => row.selected)
```

Comment:
This is where we should get rid of rows without IDs, right?

```suggestion
      .filter(row => rowId && row.selected)
```